tests/containers/import_ca_certs.pm

```diff
@@ -0,0 +1,24 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Make sure that ca certifications were installed
+# Maintainer: qa-c team <qa-c@suse.de>
+
+use Mojo::Base qw(consoletest);
+use testapi;
+use utils;
+use containers::common;
+
+sub run {
+    my ($self) = @_;
+    $self->select_serial_terminal;
+    ensure_ca_certificates_suse_installed();
```

Comment:
I'd remove also the `ensure_ca_certificates_suse_installed()` subroutine as this will become the only place where it is used..

Reply:
after some discussion with other guys i end up to implement a second alternative. if the other is found more suitable, then yes, this module is useless and then i will have to move the `ensure_ca_certificates_suse_installed()` somewhere else.